it/uk/gov/hmrc/fhregistrationfrontend/controllers/BusinessPartnersCorporateBodyUtrControllerISpec.scala

```diff
@@ -0,0 +1,98 @@
+package uk.gov.hmrc.fhregistrationfrontend.controllers
+
+import org.jsoup.Jsoup
+import play.api.libs.ws.DefaultWSCookie
+import play.api.test.WsTestClient
+import uk.gov.hmrc.fhregistrationfrontend.testsupport.{Specifications, TestConfiguration}
+
+class BusinessPartnersCorporateBodyUtrControllerISpec
+  extends Specifications with TestConfiguration {
+
+  val route = "/form/business-partners/corporate-body-corporation-tax-unique-taxpayer-reference"
+
+  s"GET $route" should {
+
+    "render the corporate-body-corporation-tax-unique-taxpayer-reference page" in {
+      given
+        .commonPrecondition
+
+      WsTestClient.withClient { client =>
+        val result = client.url(s"$baseUrl" + s"$route")
```

Comment:
Why use both string interpolation and concatenation? You could use 
`val result = client.url(baseUrl + route)` or `val result = client.url(s"$baseUrl$route")`
Doing both just adds extra steps.